### ISSUE TITLE
Diesel checks if the database is existing, which fails for Oracle

### ISSUE THREAD

**Issue by pgab** [GiGainfosystems/diesel-oci#5]
`r2d2::Builder` check constantly if the database is still available. The native syntax is `select 1`  this needs to be `select 1 from dual` in order to work for oracle.

As of right now such checks need to be disabled:
```rust
let mut builder = Pool::builder();
builder = builder.max_size(1);
// disable check on database
builder = builder.test_on_check_out(false);
```

**Comment by geomaniac** [comment]
Couldn't we do a workaround until diesel provides a way to change / specify that check query?

Something like `if mysql == "select 1" { mysql.push_str(" from dual" }`.

**Comment by pgab** [comment]
That's a good idea.

**Comment by geomaniac** [comment]
This was fixed with  #28
